test/item_manager.test.js

```diff
@@ -582,5 +582,21 @@ describe('item manager', function () {
       expect(results[2].title).to.equal(tags[3].title);
       expect(results[3].title).to.equal(tags[2].title);
     })
+  });
+
+  describe('getParentTags', async function () {
+    it('should return parent tags for a tag', async function () {
+      const parentTags = [
+        await this.itemManager.findOrCreateTagByTitle('parent.child.grandchild'),
```

Comment:
Hmm..I don't think this is right. Parent tags of `parent.child.grandchild` would be named `parent.child` and `parent`. So parent tags don't include `grandchild`.

Reply:
Oops you're right, this doesn't make sense at all. Will fix it now!